pyoos/parsers/ioos/one/describe_sensor.py

```diff
@@ -67,7 +74,7 @@ def __init__(self, element):
 
         self.shortName = self.get_ioos_def('shortName', 'identifier', ont)
         self.longName = self.get_ioos_def('longName', 'identifier', ont)
-        self.keywords = map(unicode, self.system.keywords)
+        self.keywords = list(map(str, self.system.keywords))
```

Comment:
That does not work on Python 3.

```python
>>> 'decode' in dir(str)
False
```

I guess that we need to focus more on @daf's question in the sense that it **does** change the form on Python2 because the `keywords` will no longer be `unicode`. But do we care?